### ISSUE TITLE
2.0.x-dev needs a NEW set of examples on packagist.org. Too many of them doesn't work for 2.0.x-dev

### ISSUE THREAD

**Issue by Overtonesinger** [ICanBoogie/DateTime#18]
In ver. _2.0.x-dev_ and especially in **PHP 5.6** which supports namespaces and "**use**" - here are also my workarounds to get all those examples on packagist.org working again - in this version **2.0.x-dev**:

1. new **DateTime**('now', 'Europe/Paris');  // This is an interface **now**! **Use "MutableDateTime" / "Immutable..."** instead.
2. $time->timestamp += 3600 * 4;  // The property is NOT writable anymore! Use **$time->change(array("hour" => $time->hour + 4) );**
3. echo $time->zone;  // 'LogicException' with message 'Property is not defined: zone.' ; **Use $time->tz**
4. $time = DateTime::none();  // Fatal error: Call to undefined method DateTime::none()  ;  FIX: add statement:   **use ICanBoogie\ImmutableDateTime as DateTime;**  _- somewhere before ::none()._ 
5. $now == DateTime::now()  **or**  ::right_now();  // **The same** problem.  You need to add:  **use ICanBoogie\ImmutableDateTime as DateTime;**  _- somewhere before calling ::now() or ::right_now()._

**Comment by olvlvl** [comment]
The README in packagist is extracted from `master`. Once `2.0` is merged into master the README will be updated. I believe this is the expected behavior for packagist. In the mean time, you can check the README for the `2.0` branch here: https://github.com/ICanBoogie/DateTime/tree/2.0
